Production_scripts/Process_extracts/Produce_extract_tests/extract_acute_incl_gls_tests.R

```diff
@@ -18,26 +14,24 @@ library(openxlsx)
 library(slfhelper)
 
 
-# Read in Data---------------------------------------
+# Read in Data-----------------------------------------
 
 year <- "1920"
 
+# Read new data file
+new_data <- readr::read_rds(get_source_extract_path(year, "Acute", ext = "rds"))
+
 # Read current SLF episode file and filter for 01B and GLS records
-slf_ep_1920 <- read_slf_episode("1920", recid = c("01B", "GLS")) %>%
+existing_data <- read_slf_episode("1920", recid = c("01B", "GLS")) %>%
   get_chi()
```

Comment:
Just had a thought that we probably don't need the chi number. We just use chi or anon_chi to count individual people so we could just rename anon_chi to chi and it would serve the same purpose.

```suggestion
  rename(chi = anon_chi)
```

Alternatively, you could alter the functions so that they can take chi or anon_chi.